roles/update-pkgs/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: Update packages and kernel to latest available versions
+  package:
+    name: '*'
+    state: latest
+  when: ansible_pkg_mgr in ['yum', 'dnf']
+
+- name: Check if reboot required
+  shell: needs-restarting -r
+  register: reboot_check
+  ignore_errors: yes
+  when: ansible_distribution in ['CentOS', 'RHEL']
+
+- name: Reboot if necessary
+  reboot:
+    reboot_timeout: 600
+    test_command: ping -c 4 {{ ansible_default_ipv4.address }}
```

Comment:
explanation for this field is `Command to run on the rebooted host and expect success from to determine the machine is ready for further tasks.` and the default command is `whoami`. 

So I'm not sure pining to which ip address here? can we have keep it default or have some command like `uptime` or something else for the confirmation?